x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -111,8 +114,11 @@ const UnstyledProcessEventDot = React.memo(
     const [xScale] = projectionMatrix;
 
     // Node (html id=) IDs
-    const ariaActiveDescendant = useSelector(selectors.ariaActiveDescendant);
-    const selectedNode = useSelector(selectors.selectedNode);
+    const ariaActiveDescendant = useSelector((state: ResolverState) =>
+      selectors.ariaActiveDescendant(state)
+    );
+    const selectedNode = useSelector((state: ResolverState) => selectors.selectedNode(state));
+    const originID = useSelector((state: ResolverState) => selectors.originID(state));
```

Comment:
:grey_question: At some point, maybe we should change "originID" to "analyzedEventID" to match up with the terms we're using

Reply:
Yea, I thought about doing that, but figured I'd wait till the refactor is in before I start doing any type of name changes, etc...